features/steps/migration_dir.py

```diff
@@ -14,3 +14,12 @@ def step_impl(context):
     context.migr_dir = tempfile.mkdtemp()
     os.mkdir(os.path.join(context.migr_dir, 'migrations'))
     os.mkdir(os.path.join(context.migr_dir, 'callbacks'))
+
+
+@given('no migration dir')
```

Comment:
This does not look like no migratiions dir for me. More like empty one.

Reply:
The temporary directory created here is the 'project root' which would can have any name, and contains the `migrations.yml` file. pgmigrate.py expects this folder to have a `migrations` subfolder, with exactly that name.